test/e2e/helpers.js

```diff
@@ -1087,6 +1087,16 @@ async function getCleanAppState(driver) {
   );
 }
 
+// Should execute at the beginning of a test
+async function setLocalDeveloperFlag(driver, flagName, value) {
+  return await driver.executeScript(() => {
```

Comment:
I assume the argument passed to `executeScript` is supposed to be a string?

One of the problems here is that if we set a property on window in the service worker / background, it won't be available on window in the use.